backend/eveonline/routers/characters.py

```diff
@@ -40,20 +41,27 @@ class ErrorResponse(BaseModel):
     detail: str
 
 
+class TagRequest(BaseModel):
+    character_id: int
+    tags: List[str]
+
+
 @router.get(
     "",
     summary="Get characters",
     auth=AuthBearer(),
     response=List[BasicCharacterResponse],
 )
 def get_characters(request):
-    characters = EveCharacter.objects.filter(token__user=request.user)
+    tokens = Token.objects.filter(user=request.user)
```

Comment:
The interface to eve characters is eve characters, not tokens. Need to use the query we used to have on 50 otherwise we will have "duplicate" cases (Tokens are many to one)